lua/lazyvim/plugins/extras/overseer/nvim-dap.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  "mfussenegger/nvim-dap",
+  dependencies = "stevearc/overseer.nvim",
+  opts = function()
+    require("overseer").patch_dap(true)
+    require("dap.ext.vscode").json_decode = require("overseer.json").decode
```

Comment:
Is this necessary when we already override its default json parser? https://github.com/LazyVim/LazyVim/blob/74c1766479411357054819b5d8c8376ca12c2504/lua/lazyvim/plugins/extras/dap/core.lua#L88-L93